examples/applications/plot_topics_extraction_with_nmf_lda.py

```diff
@@ -137,6 +149,65 @@ def plot_top_words(model, feature_names, n_top_words, title):
     "Topics in NMF model (generalized Kullback-Leibler divergence)",
 )
 
+# Fit the MiniBatchNMF model
+print(
+    "\n" * 2,
+    "Fitting the MiniBatchNMF model (Frobenius norm) with tf-idf "
+    "features, n_samples=%d and n_features=%d, batch_size=%d..."
+    % (n_samples, n_features, batch_size),
+)
+t0 = time()
+mbnmf = MiniBatchNMF(
```

Comment:
We get a convergence warning here. Do you think that we can tweak the parameter to avoid that?

Reply:
yes, max_iter was set to 10 for no reason and the batch size was quite high compared to the number of samples. It should be good now